alembic/dev_seeds.py

```diff
@@ -71,34 +71,34 @@ def create_source(source):
 
 create_source(
     Source(
-        id="mpv",
-        publication_name="Mapping Police Violence",
-        publication_date="01/01/2015",
-        author="Samuel Sinyangwe",
-        URL="https://mappingpoliceviolence.us",
+        id=10000000,
+        name="Mapping Police Violence",
+        url="https://mappingpoliceviolence.us",
+        contact_email="info@campaignzero.org"
     )
 )
 
 
 def create_incident(key=1, date="10-01-2019", lon=84, lat=34):
     base_id = 10000000
     id = base_id + key
+    mpv = db.session.query(Source).filter_by(name="Mapping Police Violence").first()
     incident = Incident(
         id=id,
+        source = mpv,
         location=f"Test location {key}",
         longitude=lon,
         latitude=lat,
         description=f"Test description {key}",
         department=f"Small Police Department {key}",
         time_of_incident=f"{date} 00:00:00",
-        officers=[
```

Comment:
There should probably be seeded officer data as well to be able to test the suspect_matches relationship.